test/functional/test_site/_markbind/variables.md

```diff
@@ -9,3 +9,5 @@
 <variable name="global_variable_overriding_included_variable">Global Variable Overriding Included Variable</variable>
 <variable name="global_variable">Global Variable</variable>
 <variable name="page_global_variable_overriding_page_variable">Global Variable Overriding Page Variable</variable>
+
+<variable name="base2">2019-01-01</variable>
```

Comment:
the `variables.md` filter test seems to have disappeared 👀

Reply:
Something like ` <variable name=“filtered”> {{ base2 | date(format1, 10) }}</variable> `, then using it in the page
Rationale being that the variables.md variables are resolved separately; would be good to ensure the call to njUtil is there